rocAL/rocAL/include/decoders/image/decoder.h

```diff
@@ -38,6 +38,7 @@ enum class DecoderType
     OVX_FFMPEG = 5,//!< Uses FFMPEG to decode video streams, can decode up to 4 video streams simultaneously
     FFMPEG_SOFTWARE_DECODE = 6,
     FFMPEG_HARDWARE_DECODE = 7,
+    SNDFILE = 8,//!< Uses sndfile to decode audio files
```

Comment:
Why do we need this? 
We have introduced audio_decoder.h so we should use that ryt? Please let me know where this is used.

Reply:
Its used in rocal_api_data_loaders.cpp file to evaluate audio dataset